tensorly/metrics/similarity.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+
+import numpy as np
+from sklearn.preprocessing import normalize
```

Comment:
Tensorly doesn't have sklearn as a dependency, so we can't use this. Luckily there's a `tl.norm` operation to get the norm for normalization.